x-pack/plugins/synthetics/server/legacy_uptime/routes/types.ts

```diff
@@ -61,29 +61,37 @@ export type UMKibanaRoute = UMRouteDefinition<
  * This is an abstraction over the default Kibana route type. This allows us to use custom
  * arguments in our route handlers and impelement custom middleware.
  */
-export type UptimeRoute = UMRouteDefinition<UMRouteHandler>;
+export type UptimeRoute<ClientContract = unknown> = UMRouteDefinition<
+  UMRouteHandler<ClientContract>
+>;
 
 /**
  * Functions of this type accept custom lib functions and outputs a route object.
  */
-export type UMRestApiRouteFactory = (libs: UMServerLibs) => UptimeRoute;
-export type SyntheticsRestApiRouteFactory = (libs: UMServerLibs) => SyntheticsRoute;
+export type UMRestApiRouteFactory<ClientContract = unknown> = (
+  libs: UMServerLibs
+) => UptimeRoute<ClientContract>;
+export type SyntheticsRestApiRouteFactory<ClientContract = any> = (
+  libs: UMServerLibs
+) => SyntheticsRoute<ClientContract>;
 export type SyntheticsStreamingRouteFactory = (libs: UMServerLibs) => SyntheticsStreamingRoute;
 
 /**
  * Functions of this type accept our internal route format and output a route
  * object that the Kibana platform can consume.
  */
 export type UMKibanaRouteWrapper = (
-  uptimeRoute: UptimeRoute,
+  uptimeRoute: UptimeRoute<any>,
```

Comment:
Does it work without specifying the generic arg `any` and have it default to `unknown` defined above?